### DISCUSSION TITLE
Add entire album to Queue.

### DISCUSSION THREAD

**Discussion by Francy2007** [torum/MPDCtrl#8] [Q&A]
Good morning,
i would like an answer from Toru, if possible. I can only add one file to the queue at a time. How can you add an entire album with just one command?
Sincerely.

Vincenzo

**Comment by scubanarc** [comment]
I agree that adding a full album to the queue is a needed feature.

Currently, I click on an album, then click on a single file, then press Ctrl+A to select all, then right-click and add to the queue. It works, but it's clunky.
